modules/nf-core/tiara/tiara/main.nf

```diff
@@ -0,0 +1,61 @@
+process TIARA_TIARA {
+    tag "$meta.id"
+    label 'process_medium'
+
+    // WARN: Version information not provided by tool on CLI. Please update version string below when bumping container versions.
+    conda "conda-forge::tiara=1.0.3"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/tiara:1.0.3' :
+        'quay.io/biocontainers/tiara:1.0.3' }"
+
+    input:
+    tuple val(meta), path(fasta)
+
+    output:
+    tuple val(meta), path("${prefix}.out.{txt,txt.gz}")      , emit: classifications
+    tuple val(meta), path("log_*.out.{txt,txt.gz}")                 , emit: log
+    tuple val(meta), path("*.{fasta,fasta.gz}")                     , emit: fasta, optional: true
+    path "versions.yml"                                             , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    prefix = task.ext.prefix ?: "${meta.id}"
+    VERSION = '1.0.3' // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
```

Comment:
You should also add these to the stub since the `script` clause won't be run in a stub run (and so these variables won't be created) :)